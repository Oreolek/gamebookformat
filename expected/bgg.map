{"start": 1, "second": 4, "end": 2, "third": 3}