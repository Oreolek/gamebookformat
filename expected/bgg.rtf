{\rtf1\ansi\ansicpg1252\cocoartf1038\cocoasubrtf360
{\fonttbl\f0\fswiss\fcharset0 Helvetica;}
{\colortbl;\red255\green255\blue255;}
\paperw11900\paperh16840\margl1440\margr1440\vieww14140\viewh14860\viewkind0
\pard\tx566\tx1133\tx1700\tx2267\tx2834\tx3401\tx3968\tx4535\tx5102\tx5669\tx6236\tx6803\ql\qnatural\pardirnatural

\f0\b\fs24 \cf0

\b \qc Title of Game
\b0\
\
\b \qc Introduction
\b0\



\ql  For information that goes before the actual numbered sections, if needed.  \
\
\
\b \qc More Introduction
\b0\



\ql   Any number of these optional pre-game sections are possible really. It would be useful to be able to do sub-sections here to mark up a complete rulebook as part of the paragraph booklet, but there is no such feature (yet?).  \
\

\b Turn to 1 to begin.
\b0\
\
\b \qc 1
\b0\
\ql  This is the first section (or paragraph, if you prefer). From here you can go to section \b 4
\b0
 or \b 3
\b0
.  \
\

\b \qc 2
\b0\
\ql  This is the end. There is nothing more in this gamebook. \
\

\b \qc 3
\b0\
\ql  Third section. Again, its number is not known. You can go to the end at \b 2
\b0
. This section has an image to show the syntax for doing that (same as images in bgg forums really).    \
\

\b \qc 4
\b0\
\ql  This is the second section. It may or may not end up with number 2 in the generated gamebook. From here you can skip to the \b 2
\b0
 or go to the third section at \b 3
\b0
.  \
\

}
