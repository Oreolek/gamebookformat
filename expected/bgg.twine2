<tw-storydata name="Title of Game" startnode="1" creator="Twine" creator-version="2.0" format="Harlowe" options="">
<style role="stylesheet" id="twine-user-stylesheet" type="text/twine-css"></style><script role="script" id="twine-user-script" type="text/twine-javascript"></script>
<tw-passagedata pid="1" name="1" tags="" position="10,10"> This is the first section (or paragraph, if you prefer). From here you can go to section [[4]] or [[3]]. </tw-passagedata>
<tw-passagedata pid="2" name="2" tags="" position="20,20"> This is the end. There is nothing more in this gamebook.</tw-passagedata>
<tw-passagedata pid="3" name="3" tags="" position="30,30"> Third section. Again, its number is not known. You can go to the end at [[2]]. This section has an image to show the syntax for doing that (same as images in bgg forums really). 
 </tw-passagedata>
<tw-passagedata pid="4" name="4" tags="" position="40,40"> This is the second section. It may or may not end up with number 2 in the generated gamebook. From here you can skip to the [[2]] or go to the third section at [[3]]. </tw-passagedata>
</tw-storydata>
