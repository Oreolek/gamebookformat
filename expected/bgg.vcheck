examples/bgg.json: ending (death?) section found: end
