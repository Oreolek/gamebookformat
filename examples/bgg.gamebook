title = Title of Game
author = Your Name

= Introduction
For information that goes before the actual numbered sections, if needed.

= More Introduction

Any number of these optional pre-game sections are possible really.
It would be useful to be able to do sub-sections here to mark up a
complete rulebook as part of the paragraph booklet, but there is
no such feature (yet?).

* 1 start
This is the first section (or paragraph, if you prefer). From here
you can go to section [[second]] or [[third]].

* second
This is the second section. It may or may not end up with number
2 in the generated gamebook. From here you can skip to the [[end]]
or go to the third section at [[third]].

* third
Third section. Again, its number is not known. You can go to
the end at [[end]]. This section has an image to show
the syntax for doing that (same as images in bgg forums really).
[img]testimage.png[/img]

* end
This is the end. There is nothing more in this gamebook.
