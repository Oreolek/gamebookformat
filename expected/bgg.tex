\documentclass[a5paper,onecolumn]{book}

\usepackage[utf8]{inputenc}
\usepackage[T1]{fontenc}
\usepackage[hidelinks]{hyperref}
\usepackage{graphicx}

\usepackage[top=3.3cm, bottom=3.3cm, left=2cm, right=2cm]{geometry}
\newif\ifpdf
\ifx\pdfoutput\undefined
  \pdffalse
\else
  \ifnum\pdfoutput=1
    \pdftrue
  \else
    \pdffalse
  \fi
\fi

\title{Title of Game}
\author{Your Name}
\date{}

\newcounter{sectionnr}

\begin{document}

\maketitle

\thispagestyle{empty}

\pagestyle{empty}

\clearpage

\subsection*{\begin{center} \textbf{Introduction} \end{center}}


 \noindent
 For information that goes before the actual numbered sections, if needed. 
\vspace{1em}
\subsection*{\begin{center} \textbf{More Introduction} \end{center}}


 \noindent
  Any number of these optional pre-game sections are possible really. It would be useful to be able to do sub-sections here to mark up a complete rulebook as part of the paragraph booklet, but there is no such feature (yet?). 
\vspace{1em}


Turn to 1 to begin.
\phantomsection
\refstepcounter{sectionnr}
\label{section1}
\subsection*{\begin{center} \textbf{1} \end{center}}

 \noindent
 This is the first section (or paragraph, if you prefer). From here you can go to section \textbf{\autoref{section4}} or \textbf{\autoref{section3}}. 
\vspace{1em}
\phantomsection
\refstepcounter{sectionnr}
\label{section2}
\subsection*{\begin{center} \textbf{2} \end{center}}

 \noindent
 This is the end. There is nothing more in this gamebook.
\vspace{1em}
\phantomsection
\refstepcounter{sectionnr}
\label{section3}
\subsection*{\begin{center} \textbf{3} \end{center}}

 \noindent
 Third section. Again, its number is not known. You can go to the end at \textbf{\autoref{section2}}. This section has an image to show the syntax for doing that (same as images in bgg forums really). \begin{center}
\includegraphics[width=.9\textwidth]{testimage.png}
\end{center}
 
 
\vspace{1em}
\phantomsection
\refstepcounter{sectionnr}
\label{section4}
\subsection*{\begin{center} \textbf{4} \end{center}}

 \noindent
 This is the second section. It may or may not end up with number 2 in the generated gamebook. From here you can skip to the \textbf{\autoref{section2}} or go to the third section at \textbf{\autoref{section3}}. 
\vspace{1em}
\end{document}
